src/content/blog/2023/02-09_dice-ctf.md

```diff
@@ -211,7 +211,7 @@ The encryption contains two phases.
 
 During the first one, it iterates over the vector using the global position pointer to get unique positions for each thread.
 The round counter is used as the distance between elements and also determines the operation to be used.#
```

Comment:
```suggestion
The round counter is used as the distance between elements and also determines the operation to be used.
```
I assume this `#` wasn't intentionally there?

Reply:
Yup, you're absolutely right. Thanks!